src/mlpack/methods/decision_stump/decision_stump_main.cpp

```diff
@@ -128,15 +128,12 @@ int main(int argc, char *argv[])
     mat trainingData = std::move(CLI::GetParam<mat>("training"));
 
     // Load labels, if necessary.
-    Mat<size_t> labelsIn;
+    Row<size_t> labelsIn;
     if (CLI::HasParam("labels"))
     {
-      labelsIn = std::move(CLI::GetParam<Mat<size_t>>("labels"));
+      labelsIn = std::move(CLI::GetParam<Row<size_t>>("labels"));
 
-      // Do the labels need to be transposed?
-      if (labelsIn.n_cols == 1)
-        labelsIn = labelsIn.t();
-      if (labelsIn.n_cols == 1)
+      if (labelsIn.n_rows > 1)
         Log::Fatal << "Labels must be one-dimensional!" << endl;
```

Comment:
This check isn't needed anymore, because `labelsIn` can't possibly have more than one row now.  So we can simplify this code.  Later on, when `labelsIn` is used (I believe it's usually used as `labelsIn.row(0)`), that can be simplified to just `labelsIn`.  We can even change the name of the variable to simply `labels`.